yt/utilities/lib/bounding_volume_hierarchy.pyx

```diff
@@ -1,7 +1,8 @@
 # distutils: libraries = STD_LIBS
 # distutils: include_dirs = LIB_DIR
-# distutils: extra_compile_args = OMP_ARGS
-# distutils: extra_link_args = OMP_ARGS
+# distutils: language = c++
+# distutils: extra_compile_args = CPP14_FLAG OMP_ARGS
+# distutils: extra_link_args = CPP14_FLAG OMP_ARGS
```

Comment:
isn't this meant for #2939 ?

Reply:
I think it's fine since this is high priority and will I hope we merge #2939 asap too !